example-deployment/keycloak-tools/bootstrap-keycloak.js

```diff
@@ -97,7 +97,69 @@ function authHeaders(token) {
         eventsListeners: ['jsonlog_event_listener'],
       },
     );
+
+    // create audience client scope
+    await request(
+      `${baseUrl}/admin/realms/rmf-web/client-scopes`,
+      {
+        method: 'POST',
+        headers: authHeaders(token),
+      },
+      {
+        name: 'dashboard',
+	protocol: 'openid-connect',
+	description: 'dashboard scope',
+	protocolMappers: [
+	  {
+	    config: {
+	      "access.token.claim": "true",
+	      "id.token.claim": "false",
+	      "included.client.audience": "dashboard"
```

Comment:
Following auth0, I think the aud should be a url.

Reply:
These exact values were sniffed from the web inspector when i sent the request using the KeyCloak Admin panel, but i can try out the url form for `included.client.audience` and see if that works as well

Reply:
i briefly tried using ` "included.client.audience": ${baseUrl}/admin/realms/rmf-web/clients/dashboard` but it seems to "not work", but i'm not sure if i'm doing misunderstanding the keycloak api?

Reply:
Did you change the aud in the api-server to match the new settings?

Reply:
> Did you change the aud in the api-server to match the new settings?

It's already set on `config["aud"] = "dashboard"` on rmf_server_config.py or do you mean to change it on the `default_config.py` file?

> Following auth0, I think the aud should be a URL.


Auth0 supports both ways. You can see an example here https://auth0.com/docs/tokens#access-tokens. I tested with the URL as well but it does not seem to be working, most of the `aud` examples on Keycloak set just a name on the `aud`, the documentation it's not very helpful https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_protocolmapperrepresentation 
I guess we should stick to that for now since Auth0 supports this as well.